lib/kaiser/command_runner.rb

```diff
@@ -19,6 +19,14 @@ def self.run!(out, cmd, &block)
     def initialize(out, cmd)
       @out = out
       @cmd = cmd.tr "\n", ' '
+
+      if @cmd.start_with? 'docker run' && arm?
+        @cmd.sub! /\A(docker run)/, '\1 --platform linux/amd64,linux/arm64/'
```

Comment:
@davidsiaw I managed to set up MySQL successfully on my arm mac after tweaking a little like below, but got stuck at the step to set up the app due to docker unable to find a `kaiser:hats-master` locally (image attached). I think something went wrong during [this command](https://github.com/degica/kaiser/blob/4bbce0e300f78221bf8e87cf459adf08c9781d1d/lib/kaiser/cmds/up.rb#L34-L38) to build the image but not sure why. Do you have any ideas?
```suggestion
      if @cmd.start_with?('docker run') && arm? # No implicit conversion of String to Boolean error
        @cmd.sub! /\A(docker run)/, '\1 --platform linux/amd64'  # command line seems unable to read 2 arguments for --platform flag. It complained about 'amd64,linux' not being a valid argument
        # using `amd64` as suggested in docker docs https://docs.docker.com/desktop/mac/apple-silicon/
```
<img width="1268" alt="Screen Shot 2022-07-05 at 16 21 11" src="https://user-images.githubusercontent.com/53965573/177272263-33a6bf32-efd4-4f32-883e-3310006c84fd.png">

Reply:
the lack of the image means the error us actually further up